charts/alluxio/templates/fuse/daemonset.yaml

```diff
@@ -108,6 +108,12 @@ spec:
                 fieldPath: status.hostIP
           - name: ALLUXIO_CLIENT_JAVA_OPTS
             value: " -Dalluxio.user.hostname=${ALLUXIO_CLIENT_HOSTNAME} "
+          - name: FLUID_RUNTIME_TYPE
```

Comment:
Please update chart's change log and version.

Reply:
Done